broker/test/test_broker/mocks/index.js

```diff
@@ -44,8 +44,20 @@ function init() {
   return exports;
 }
 
-function verify() {
+function verify(ignoreList = []) {
```

Comment:
Add a single line comment to explain  purpose, format of the `ignoreList` variable